src/core/console_user_server/include/configuration_manager.hpp

```diff
@@ -47,6 +47,11 @@ class configuration_manager final {
       grabber_client_.add_simple_modification(pair.first, pair.second);
     }
 
+    logger_.info("loading compound modifications");
+    for (const auto& pair : configuration_core_->get_current_profile_compound_modifications()) {
+      logger_.info("found compound keys for {0}", (uint32_t) pair.first);
```

Comment:
This will be something like `grabber_clien_.add_compound_modification(pair.first, pair.second);`